### DISCUSSION TITLE
filter  olx.co.id

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1468] [Personal Filter]
![image](https://github.com/realodix/AdBlockID/assets/19517680/1ab8d019-1071-4220-8477-3e1bc09822c5)

bro @realodix bisa minta tolong bikin kan filter untuk hide kotak merah seperti di SS

terimakasih

**Comment by realodix** [comment]
Sedikit ngeri, berusaha sedetail mungkin karena takut kena halaman yg lain.. Klo ada yg bisa improve, bisa ditambahkan.. 

```adblock
olx.co.id##ul[data-aut-id="itemsList"] > li:has(button:has-text(/pasang iklan/i))
```

![noname-crop2](https://github.com/realodix/AdBlockID/assets/1314456/eb72447b-49c6-4e7d-889f-be04ccc9c634)

```adblock
olx.co.id##ul[data-aut-id="itemsList"] > li[data-aut-id="itemBox"]:has(label:has-text(/highlight/i))
```

![noname-crop](https://github.com/realodix/AdBlockID/assets/1314456/86573ebf-a3ff-4f51-852c-0b98fe17a4ef)
